manifests/server.pp

```diff
@@ -78,6 +78,7 @@
   Boolean $handle_creds                                = $mongodb::params::handle_creds,
   Boolean $store_creds                                 = $mongodb::params::store_creds,
   Array $admin_roles                                   = $mongodb::params::admin_roles,
+  Optional[String] $version                            = $facts['mongodb_version'],
```

Comment:
I don't think this is needed

Reply:
I'm happy to remove that but I'm unable to test as I don't have access to that cluster any more.